src/components/telemetry_monitor/test/telemetry_monitor_test.cc

```diff
@@ -90,14 +90,14 @@ TEST(TelemetryMonitorTest, MessageProcess) {
   EXPECT_CALL(am_observeble, SetTelemetryObserver(_));
   EXPECT_CALL(transport_manager_mock, SetTelemetryObserver(_));
   telemetry_monitor::TelemetryMonitor telemetry_monitor(server_address, port);
-  StreamerMock streamer_mock(&telemetry_monitor);
+  StreamerMock* streamer_mock = new StreamerMock(&telemetry_monitor);
```

Comment:
@pold500, sorry, but comment on the next line says
`// streamer_mock will be freed by telemetry_monitor on destruction`